src/aggregation/low_count.c

```diff
@@ -14,11 +14,11 @@ typedef struct AidResult
   bool low_count;
 } AidResult;
 
-static AidResult calculate_aid_result(seed_t bucket_seed, const AidTrackerState *tracker)
+static AidResult calculate_aid_result(const AidTrackerState *tracker)
 {
   AidResult result = {.aid_seed = tracker->aid_seed};
 
-  seed_t seeds[] = {bucket_seed, tracker->aid_seed};
+  seed_t seeds[] = {tracker->aid_seed};
```

Comment:
I presume, you're referring to `count_distinct.c`. However, are we sure that needs the same change? The specific attack we're tackling is for the LCF per se. In `count_distinct` case I think we won't be able to fully de-noise the result (as it was the case with low-count filtering), since the AID layer is still there.

If we drop `bucket_seed` from `count_distinct`, it seems like we should drop it everywhere (from noisy counts as well), since manipulating the SQL generalizations has the same effect there.

Reply:
`count(distinct)` needs to know which values are low-count and which are high-count. The mechanism has to be consistent with the way buckets are treated generally. So yes, it needs to be updated as well.